pkg/config/setup/config.go

```diff
@@ -2521,6 +2543,14 @@ func bindEnvAndSetLogsConfigKeys(config pkgconfigmodel.Setup, prefix string) {
 	config.BindEnv(prefix + "logs_dd_url") // Send the logs to a proxy. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
 	config.BindEnv(prefix + "dd_url")
 	config.BindEnv(prefix + "additional_endpoints")
+	config.ParseEnvAsSlice(prefix+"additional_endpoints", func(in string) []interface{} {
+		var mappings []interface{}
+		if err := json.Unmarshal([]byte(in), &mappings); err != nil {
+			log.Errorf(`"%s" can not be parsed: %v`, prefix+"additional_endpoints", err)
+		}
+		return mappings
+	})
```

Comment:
nit + personal opinion + no-blocker: should we make it into a helper function, same as `enableJSONParsingFromEnv`? I know. It is used only in one place, but it would make the file more homogeneous